docs/api/connection.rst

```diff
@@ -179,6 +179,19 @@ Connection
         This method takes :js:class:`optional query arguments
         <AwaitConnection>`.
 
+        .. note::
+
+            Caution is advised when reading ``decimal`` or ``bigint``
+            values using this method. The JSON specification does not
+            have a limit on significant digits, so a ``decimal`` or a
+            ``bigint`` number can be losslessly represented in JSON.
+            However, JSON decoders in JavaScript will often read all
+            such numbers as ``number`` values, which may result in
+            precision loss. If such loss is unacceptable, then
+            consider casting the value into ``str`` and decoding it on
+            the client side into a more appropriate type, such as
+            ``BigInt``.
```

Comment:
Can you make the BigInt a hyperlink to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt?

Reply:
Done